libtsuba/src/RDGTopology.cpp

```diff
@@ -273,21 +273,7 @@ katana::RDGTopology::MapMetadataExtract(
       "Extracted num_edges = {} does not match the known num_edges = {}",
       num_edges_, num_edges);
 
-  topology_state_ = TopologyKind::kCSR;
```

Comment:
Even with your change, these lines:
```
topology_state_ = TopologyKind::kCSR;
  transpose_state_ = TransposeKind::kNo;
  edge_sort_state_ = EdgeSortKind::kAny;
  node_sort_state_ = NodeSortKind::kAny;
```
must still be present here as they are setting state in the RDGTopology object itself here.

Reply:
Oops! You are correct.